runtime/src/actor_error.rs

```diff
@@ -159,6 +159,22 @@ macro_rules! actor_error {
     };
 }
 
+/// Convenience macro for generating Actor Errors
+#[macro_export]
+macro_rules! decode_params {
```

Comment:
Too late, but this doesn't belong in this file... And the documentation is wrong.

(let's fix this on master and do another merge later)

Reply:
https://github.com/filecoin-project/builtin-actors/pull/969